x/farming/keeper/staking.go

```diff
@@ -224,15 +224,24 @@ func (k Keeper) Stake(ctx sdk.Context, farmerAcc sdk.AccAddress, amount sdk.Coin
 		return err
 	}
 
+	numStakingCoinDenoms := 0
 	for _, coin := range amount {
 		queuedStaking, found := k.GetQueuedStaking(ctx, coin.Denom, farmerAcc)
 		if !found {
 			queuedStaking.Amount = sdk.ZeroInt()
 		}
 		queuedStaking.Amount = queuedStaking.Amount.Add(coin.Amount)
 		k.SetQueuedStaking(ctx, coin.Denom, farmerAcc, queuedStaking)
+
+		_, found = k.GetStaking(ctx, coin.Denom, farmerAcc)
+		if found {
+			numStakingCoinDenoms++
+		}
 	}
 
+	params := k.GetParams(ctx)
```

Comment:
Sorry for additional review, 
How about 
```
	if numStakingCoinDenoms > 0 {
		params := k.GetParams(ctx)
		ctx.GasMeter().ConsumeGas(sdk.Gas(numStakingCoinDenoms)*params.DelayedStakingGasFee, "DelayedStakingGasFee")
	}
```

and could you add comments for a link or description of DelayedStakingGasFee